djstripe/management/commands/djstripe_process_failed_events.py

```diff
@@ -0,0 +1,74 @@
+from django.core.management.base import BaseCommand
+
+from ... import models
+from ._djstripe_command_utils import VerbosityAwareOutputUtils
+
+
+class Command(BaseCommand, VerbosityAwareOutputUtils):
```

Comment:
Shouldn't the mixin be on the left here?

Reply:
Should it? Does that matter somehow?

Reply:
Yeah, since the classes are evaluated right to left, so the left overrides properties of the right, so it's good practice.

I'd also suggest putting Mixin at the end the classname.

Reply:
Ah, that makes sense. I learn something new every day. I'll implement this change.